pom.xml

```diff
@@ -104,6 +104,21 @@
             <artifactId>kafka-connect-storage-wal</artifactId>
             <version>${confluent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.esotericsoftware.kryo</groupId>
+            <artifactId>kryo</artifactId>
+            <version>2.22</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro-mapred</artifactId>
+            <version>1.8.2</version>
```

Comment:
Should we use Maven properties for these versions?

Reply:
yes, good point. since these are exclusively here to support hive, should we prefix the variables with `hive-` (for example, hive-lang3.version)?

Reply:
How about `${hive.commons.lang3.version}` and `${hive.kyro.version}`, and then `${avro.version}` which is the same that [Schema Registry uses]() but which isn't in any of the parent POMs for this repo. 

So, we can either duplicate `${avro.version}` here (risking divergence from SR), or we can instead put the `${avro.version}` Maven property in the common parent POM and have Schema Registry and this repo both inherit that property value. Any thoughts, @ewencp and @kkonstantine?

Reply:
I would prefer to reuse avro.version as well, but it doesn't look like we can inherit it as we depend on avro-connector, and not schema registry. Any thoughts on what can be done here?

Reply:
We inherit properties via parent POMs, which means we could change [Common's POM](https://github.com/confluentinc/common/blob/master/pom.xml) to add the property, and then we would inherit it since [Storage Common inherits from Common](https://github.com/confluentinc/kafka-connect-storage-common/blob/master/pom.xml#L25-L27). Then, the [Schema Registry parent POM](https://github.com/confluentinc/schema-registry/blob/master/pom.xml) would also inherit it (via REST Utils that inherits from Common), and the SR and Avro Converters all get it that way as well.

Reply:
I'm not enthusiastic w.r.t prefixing with the package that is requiring this dependency. 

This information can and should be extracted reliably via 
`mvn dependency:tree -Dverbose`

If another package needs the same dependency in the future, either the prefix will seem strange or another property with a prefix will be required. This doesn't sound ideal.

Reply:
good point, @kkonstantine. But the dependency tree will not show a dependency between hive and kryo (for example) because hive-exec doesn't explicitly declare a dependency to kryo (that's why we need to add this ourselves). If you look over the dependencies, kryo and commons-lang will look like its not being used anywhere in the code (and might get incorrectly removed).